web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2023 Jarvus Innovations LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mobilitydata.gtfsvalidator.web.service.controller;
+
+import com.google.cloud.storage.*;
+import java.io.*;
+import java.net.URL;
+import java.util.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import org.mobilitydata.gtfsvalidator.web.service.util.StorageHelper;
+import org.mobilitydata.gtfsvalidator.web.service.util.ValidationHandler;
+import org.mobilitydata.gtfsvalidator.web.service.util.ValidationJobMetaData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+public class ValidationController {
+
+  private final Logger logger = LoggerFactory.getLogger(ValidationController.class);
+
+  @Getter(AccessLevel.PROTECTED)
+  @Setter(AccessLevel.PROTECTED)
+  @Autowired
+  private Storage storage;
+
+  @Autowired private ApplicationContext applicationContext;
+
+  /**
+   * Creates a new job id and returns it to the client. If a url is provided, the file is downloaded
+   * from the url and saved to GCS. If no url is provided, a unique url is generated for the client
+   * to upload the GTFS file.
+   */
+  @CrossOrigin(origins = "*")
+  @PostMapping(value = "/create-job", produces = "application/json", consumes = "application/json")
+  public CreateJobResponse createJob(@RequestBody CreateJobRequest body) {
+    try {
+      final var jobId = UUID.randomUUID().toString();
+      var countryCode = "";
+      StorageHelper storageHelper = new StorageHelper(storage, applicationContext);
+
+      if (body != null && !body.getCountryCode().isEmpty()) {
+        countryCode = body.getCountryCode();
+        storageHelper.saveJobMetaData(jobId, countryCode);
```

Comment:
consider getting ride of `countryCode` variable and just inlining the usage here